ext/drx/drx.h

```diff
@@ -159,8 +160,10 @@ DR_EXPORT
  * \note The counter update is racy (i.e., not synchronized among threads)
  * unless #DRX_COUNTER_LOCK is specified in \p flags. When #DRX_COUNTER_LOCK
  * is set, the instrumentation may fail if a 64-bit counter is updated in
- * a 32-bit application or the counter crosses cache lines. Currently,
- * #DRX_COUNTER_LOCK is not yet supported on ARM.
+ * a 32-bit application or the counter crosses cache lines. Currently, #DRX_COUNTER_LOCK
+ * is not yet supported on ARM. For ARM, if #DRX_COUNTER_REL_ACQ is specified in \p flags,
```

Comment:
Maybe best to s/ARM/AArchXX/ so it's clear it's both 32-bit and 64-bit ("ARM" means just 32-bit in many contexts, though not in others so it can be ambiguous)?

Reply:
Done.